microsoft-edge/extensions-chromium/developer-guide/declare-permissions.md

```diff
@@ -44,7 +44,7 @@ The following table lists the currently available permission strings to use in y
 |:--- |:--- |
 | `activeTab` | Requests that the extension is granted permissions according to the `activeTab` specification. |
 | `alarms` | Gives your extension access to the `chrome.alarms` API. |
-| `background` | Makes Microsoft Edge start up early and shut down late, so that extensions may have a longer life.  When any installed extension has `background` permission, Microsoft Edge runs invisibly as soon as the user logs into the user's computer, and before the user launches Microsoft Edge.  The `background` permission also makes Microsoft Edge continue running, even after its last window is closed, until the user explicitly quits Microsoft Edge.  This permission doesn't affect extensions that are turned off in the browser.  The `background` permission is normally used on a background page. |
+| `background` | Makes Microsoft Edge start up early and shut down late, so that extensions may have a longer life.  When any installed extension has `background` permission, Microsoft Edge runs invisibly as soon as the user logs into the user's computer, and before the user launches Microsoft Edge.  The `background` permission also makes Microsoft Edge continue running, even after its last window is closed, until the user explicitly quits Microsoft Edge. Note: Disabled extensions are treated as if they aren't installed. The `background` permission is normally used on a background page. |
```

Comment:
```suggestion
| `background` | Makes Microsoft Edge start up early and shut down late, so that extensions may have a longer life.  When any installed extension has `background` permission, Microsoft Edge runs invisibly as soon as the user logs into the user's computer, and before the user launches Microsoft Edge.  The `background` permission also makes Microsoft Edge continue running, even after its last window is closed, until the user explicitly quits Microsoft Edge. The `background` permission is normally used on a background page. **Note:** Disabled extensions are treated as if they aren't installed. |
```